### PR TITLE
release: v1.7.0 [MTT-9035]

### PR DESCRIPTION
**Pull request by RikuTheFuffs**
### Description
PR for v1.7.0 release.

### Issue Number(s)
- https://jira.unity3d.com/browse/MTT-9035

### Contribution checklist
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.7.0] 2024-08-31
```
Comment (RikuTheFuffs) [inline CHANGELOG.md:3]: @Elfi0Kuhndorf / @fernando-cortez do you want to add something about the DA sample here?

**Comment by RikuTheFuffs** [comment]
@harshala-rajesh it seems that tests are not triggering for this PR, even though both the general repository and a few of the samples have been touched (MultiplayerUseCases , DedicatedGameServer).

Do you have any idea of why that could be? (I have a feeling it's related to what was done in this https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/195 )

**Comment by harshala-rajesh** [comment]
@RikuTheFuffs for some reason, though we had merged the changes to main, main does not have any changes done in https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/195, Only develop has the changes

**Comment by RikuTheFuffs** [comment]
> @RikuTheFuffs for some reason, though we had merged the changes to main, main does not have any changes done in #195, Only develop has the changes

Thanks for having a look! Are you saying that once we merge this PR, then everything should start working again?

I'm not sure it'll be enough, because I made a couple test branches towards `develop` to see if tests were going to run when adding a text file to a sample, and it seems they didn't (see #215 and #214 )

**Comment by RikuTheFuffs** [comment]
Tried again with this PR ( https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/216 ), adding just a text file for every project, it doesn't trigger anything even if it targets develop :(

**Comment by RikuTheFuffs** [comment]
I also opened a thread in the [Yamato channel](https://unity.slack.com/archives/C998PJJDD/p1724947455252689) to get further help.

**Comment by RikuTheFuffs** [comment]
Tests are running and succeeded! It looks like we had to add a point of contact in Yamato's project. They should also run again for other branches now.

**Review by fernando-cortez — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (RikuTheFuffs) [reply CHANGELOG.md:3]: Fernando answered with "nope" on slack, resolving

**Pull request merged**